src/main/java/codesquad/springcafe/model/Article.java

```diff
@@ -0,0 +1,57 @@
+package codesquad.springcafe.model;
+
+import codesquad.springcafe.dto.ArticleRequestDto;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Article {
+
+    private final Long articleId;
+    private final String writer;
+    private final String title;
+    private final String contents;
+    private final LocalDateTime localDateTime;
+    private int hits;
+
+    public Article(Long articleId, String writer, String title, String contents) {
+        this.articleId = articleId;
+        this.writer = writer;
+        this.title = title;
+        this.contents = contents;
+        this.localDateTime = LocalDateTime.now();
+        this.hits = 0;
+    }
+
+    public Article(long articleId, ArticleRequestDto articleRequestDto) {
+        this(articleId, articleRequestDto.getWriter(), articleRequestDto.getTitle(), articleRequestDto.getContents());
+    }
+
+    public Long getArticleId() {
+        return articleId;
+    }
+
+    public String getWriter() {
+        return writer;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public String getContents() {
+        return contents;
+    }
+
+    public String getLocalDateTime() {
```

Comment:
진짜 getter 가 아니기 때문에 이름을 다르게 주는 게 좋아요.
```suggestion
    public String getFormattedLocalDateTime() {
```